app/page.tsx

```diff
@@ -22,7 +22,7 @@ export default async function Home({
   const jsonData = await getCmsData();
 
   return (
-    <div className="flex flex-col items-center">
+    <div className="flex flex-col items-center w-[min(100%,680px)]">
```

Comment:
min-width classes didn’t help here?

something like `min-w-[680px] w-full` should handle this

Reply:
https://developer.mozilla.org/en-US/docs/Web/CSS/min

I much prefer this as it's a single line and easier to understand. This basically says use the smaller of these values so is the same as what you're suggesting without the complexity of the cascade

Reply:
ah so you want to have a fixed width element only when it is <680px and then arbitrary size when >680px?

i wonder if we are better off using some of tailwind's breakpoints to set these specific widths at different sizes?

Reply:
I copied the values from the builder i believe but happy to iterate on it

Reply:
yeah i'm not pleased with how the builder handles responsiveness personally. i think that doing fixed width at each breakpoint might be the move to keep things more manageable going forwards

Reply:
Adjusted the styles for the grid builder, let me know what you think!